src/ui/server/shared-validation/alpha-characters-hyphen-and-apostrophe-only/index.test.ts

```diff
@@ -0,0 +1,26 @@
+import alphaCharactersHyphenAndApostropheOnlyValidation from '.';
+import regexValidation from '../regex-validation';
+import { mockErrors, mockErrorMessagesObject } from '../../test-mocks';
+import { REGEX } from '../../constants';
+
+describe('shared-validation/alpha-characters-hyphen-and-apostrophe-only', () => {
+  const mockFieldId = 'Mock field';
+
+  describe('when the value has special characters', () => {
```

Comment:
I don't think we need the `describe` here as the function just calls `regexValidation` regardless?